### PR TITLE
Initial

### PR DESCRIPTION
**Pull request by jquense**
tired of rewriting this component everywhere

### CODE REVIEW

File: src/css-modules/Layout.module.css [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+}
+.justify-space-between {
+  justify-content: space-between;
+}
+.justify-space-around {
+  justify-content: space-around;
+}
+
+.pad-1.row > * + * {
+  margin-left: var(--pad-1);
```
Comment (jquense) [inline src/css-modules/Layout.module.css]: probably should have a solution here that supports more browsers (tho this is covered by the bfly browser matrix at the moment)

File: www/.cache/api-runner-browser.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+"use strict";
```
Comment (taion) [inline www/.cache/api-runner-browser.js]: what is this `.cache`?

File: src/css-modules/Block.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    justify: _3,
+    ...props
+  },
+  ref,
+) {
+  return (
+    <Component
+      {...props}
+      ref={ref}
+      style={{ ...style, flex: flex === true ? 1 : flex }}
```
Comment (taion) [inline src/css-modules/Block.js]: could make for some weirdness with pseudo whatever in CSS?

File: src/css-modules/Layout.module.css [hunk 4]
```diff
@@ -0,0 +1,135 @@
+.block {
+  display: block;
+}
+.block-inline {
+  display: inline-block;
+}
+
+.flex {
+  --pad-1: 2px;
```
Comment (taion) [inline src/css-modules/Layout.module.css]: can this be overridden from the outside?

File: src/css-modules/camelCase.js [hunk 5]
```diff
@@ -0,0 +1 @@
+export default str => str.replace(/[_.-](\w)/g, (_, x) => x.toUpperCase());
```
Comment (taion) [inline src/css-modules/camelCase.js:1]: we already have lodash though :p

File: package.json [hunk 6]
```diff
@@ -1,9 +1,8 @@
 {
   "name": "@4c/layout",
-  "version": "1.0.0",
-  "main": "lib/index.js",
-  "modules": "es/index.js",
-  "jsnext:main": "es/index.js",
+  "version": "0.0.0-development",
+  "main": "lib/css-modules/index.js",
```
Comment (taion) [inline package.json:4]: why the namespacing?

File: www/.cache/api-runner-browser.js (hunk 2 above)
Comment (jquense) [reply www/.cache/api-runner-browser.js]: ugh its a gatsby build artifact

File: src/css-modules/camelCase.js (hunk 5 above)
Comment (jquense) [reply src/css-modules/camelCase.js:1]: where? do we? this was simple :P

File: package.json (hunk 6 above)
Comment (jquense) [reply package.json:4]: in my head there may be multiple versions per styling target here, like a Sass only implementation

Comment (jquense) [reply package.json:4]: or styled component idk

Comment (taion) [reply package.json:4]: fair enough

File: src/css-modules/camelCase.js (hunk 5 above)
Comment (taion) [reply src/css-modules/camelCase.js:1]: i guess not in this lib :p

**Review by taion — APPROVED** [review]
gitignore the gatsby stuff?

**Pull request merged**